### DISCUSSION TITLE
Does this work for props also, or just characters?

### DISCUSSION THREAD

**Discussion by DataJuggler** [soupday/cc3_blender_tools#127] [Q&A] [answered]
I guess I will install and find out

Thanks

**Comment by DataJuggler** [comment]
It looks like this is only for characters. Characters are on my to do list to learn.

**Comment by soupday** [comment, marked as answer]
It should work for props. At least it did.

https://www.youtube.com/watch?v=tbeJUGVEhvg

https://www.youtube.com/watch?v=wizgZ1lenE0

Not so sure about animated props though...

**Comment by DataJuggler** [comment]
OK, I will watch the video here in a second. Thank you.

**Comment by DataJuggler** [comment]
I didn't notice the Prop tab. Thanks.
